runners/spark/src/main/java/org/apache/beam/runners/spark/util/BroadcastHelper.java

```diff
@@ -77,8 +77,8 @@ public synchronized T getValue() {
     }
 
     @Override
-    public void broadcast(JavaSparkContext jsc) {
-      this.bcast = jsc.broadcast(value);
+    public synchronized void broadcast(JavaSparkContext jsc) {
+      this.bcast = jsc.broadcast(getValue());
```

Comment:
It seems that synchronizing the `broadcast` methods is enough to satisfy findbugs.